test_runner/src/test/kotlin/ftl/reports/output/OutputReportLoggersTest.kt

```diff
@@ -74,9 +74,23 @@ class OutputReportLoggersTest {
         val total: BigDecimal = BigDecimal.ZERO
 
         // when
-        outputReport.log(physical, virtual, total)
+        outputReport.logCosts(physical, virtual, total)
 
         // then
         assertThat(outputReport.outputData).containsKey("cost")
     }
+
+    @Test
+    fun `should log billable minutes`() {
+        // given
+        val physical: BigDecimal = BigDecimal.ONE
+        val virtual: BigDecimal = BigDecimal.TEN
+        val total: BigDecimal = BigDecimal.ZERO
+
+        // when
+        outputReport.logBillableMinutes(physical, virtual, total)
+
+        // then
+        assertThat(outputReport.outputData).containsKey("billable-minutes")
```

Comment:
Is it worth asserting anything about the keys/values of the new billable minutes structure? I see that the test above does a similar check only looking for the top level key. It seems like the intention is to feed this output to another program/process, so defining the entire contract here in the test would help ensure it can't be broken in the future.